pkg/argo/fetch_test.go

```diff
@@ -62,6 +63,19 @@ spec:
       command: [echo]
       args: ["{{inputs.parameters.message}}"]
 `
+
+	podYAML = `apiVersion: v1
+kind: Pod
+metadata:
+  name: sqlflow-pod
+spec:
+  restartPolicy: Never
+  containers:
+  - name: main 
+    image: docker/whalesay
+    command: [bash]
+    args: [-c, "echo 'hello1\nhello2'; sleep 2; echo 'hello3'"]
```

Comment:
Maybe change to `bash -c 'for i in {0..100}; do   echo $i;   sleep 0.0$((RANDOM % 100)); done'` for more intensive testing.

Reply:
Good point, I have tested it on my laptop and it passed, will do that in the next PR.